libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/map/MapViewBinderTest.kt

```diff
@@ -164,8 +165,14 @@ class MapViewBinderTest {
 
         val firstComponent = components.findComponent { it is LocationPuckComponent }
         navContext.applyStyleCustomization {
-            locationPuck =
-                LocationPuck2D(bearingImage = ctx.getDrawable(android.R.drawable.arrow_down_float))
+            locationPuckOptions = LocationPuckOptions
+                .Builder(ctx)
+                .freeDrivePuck(
```

Comment:
How can developer set a custom puck for all navigation states?
Does he have to set the same puck for each state?

Reply:
Do you mean set the same puck for all navigation states? In that case they would have to set the same puck to each state.

Reply:
Addressed as suggested [here](https://github.com/mapbox/mapbox-navigation-android/pull/6574#discussion_r1018273540)